Extension/lib/content-script/preload.js

```diff
@@ -43,6 +43,7 @@
     var isFirefox = false;
     var isOpera = false;
     var shadowRoot = null;
+    var loadTruncatedCss = false;

```

Comment:
We need an explanation comment here. Otherwise it is not clear enough WHY we do it.